components/molecule/textareaField/src/index.js

```diff
@@ -19,6 +19,11 @@ const MoleculeTextareaField = WithCharacterCount(
     onChange,
     ...props
   }) => {
+    const hasErrors = (success, error) => {
+      if (error) return true
+      if (success) return false
+    }
```

Comment:
are you sure, that you need this function ?!

Reply:
From actual implementation in [molecule/select](https://github.com/SUI-Components/sui-components/blob/master/components/molecule/select/src/index.js), I guess that if error is true it doesn't mean that success is false and viceversa..